src/App.js

```diff
@@ -14,11 +15,87 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    isStarted: false,
+    isReversed: false,
+    sortBy: '',
+    goods: goodsFromServer,
+  }
+
+  start = () => {
+    this.setState({ isStarted: true });
+  }
+
+  reverse = () => {
```

Comment:
`reverse` what? 
```suggestion
  reverseHandler = () => {
```
это касается и остальных методов